src/test/java/org/fcrepo/importexport/importer/ImporterTest.java

```diff
@@ -144,8 +165,19 @@ public void testImportBinary() throws Exception {
     }
 
     @Test
+    public void testImportExternalResource() throws Exception {
+        final Importer importer = new Importer(externalResourceArgs, clientBuilder);
+        importer.run();
+        verify(client).put(externalResourceURI);
+        verify(externalResourceBuilder).body(eq(new File(externalFilesDir, "rest/ext1.external")),
+                                eq("message/external-body"));
+        verify(client).put(externalResourceDescriptionURI);
+    }
+
+    @Test
     public void testImportWithoutBinaries() throws Exception {
-        final Importer importer = new Importer(noBinaryArgs, clientBuilder);
+        binaryArgs.setIncludeBinaries(false);
```

Comment:
For clarity I'm happy to use a copy of the config.  But in terms of order,  it shouldn't matter considering the binaryArgs flag gets set to true in the setup() method which runs before each test.  But you're right, it could open the door for confusion down the road.